diff --git a/flecsi/topo/narray/interface.hh b/flecsi/topo/narray/interface.hh
index 3c84f87b0..f86e3681f 100644
--- a/flecsi/topo/narray/interface.hh
+++ b/flecsi/topo/narray/interface.hh
@@ -762,16 +762,16 @@ protected:
     }
   }
 
+  template<index_space S, axis A>
+  FLECSI_INLINE_TARGET const axis_color & get_axis() const {
+    return meta_->template get<S>().axcol.template get<A>();
+  }
+
 private:
   template<axis A>
   FLECSI_TARGET static constexpr std::uint32_t to_idx() {
     return axes::template index<A>;
   }
-
-  template<index_space S, axis A>
-  FLECSI_INLINE_TARGET const axis_color & get_axis() const {
-    return meta_->template get<S>().axcol.template get<A>();
-  }
 }; // struct narray<Policy>::access
 
 /*----------------------------------------------------------------------------*
